source/adapter.screenshare.js

```diff
@@ -141,6 +141,10 @@
 
     getUserMedia = navigator.getUserMedia;
 
+  } else if (window.webrtcDetectedBrowser === 'edge') {
```

Comment:
Ok. I'm not particular how the edge should be detected as long as it works in most environments :)